cfgov/v1/wagtail_hooks.py

```diff
@@ -110,3 +113,57 @@ def register_django_admin_menu_item():
         classnames='icon icon-redirect',
         order=99999
     )
+
+
+class RelativePageLinkHandler(PageLinkHandler):
+    """
+    Rich text link handler that forces all page links to be relative.
+
+    This special page link handler makes it so that any internal Wagtail page
+    links inserted into rich text fields, for example:
+
+        <a linktype="page" id="123">foo</a>
+
+    always get replaced with a relative URL to that page, never an absolute
+    one, like this:
+
+        <a href="/path/to/page">foo</a>
+
+    In standard Wagtail behavior, this may be an absolute URL if an
+    installation has multiple Wagtail Sites. In our current custom usage we
+    have multiple Wagtail Sites (one for production, one for staging) that
+    share the same root page. So forcing the use of relative URLs would work
+    fine and allow for easier navigation within a single domain.
+
+    This will explicitly break things if users ever wanted to host some
+    additional site that doesn't share the same root page.
+
+    This code is modified from `wagtail.wagtailcore.rich_text.PageLinkHandler`.
+
```

Comment:
Trivial: You could remove this blank line to match our typical docstring formatting